src/map.cpp

```diff
@@ -4259,11 +4259,40 @@ unsigned int temp_difference_ratio( int temp_one, int temp_two )
     return ratio;
 }
 
+int check_freezing_temperature( item &it )
+{
+    itype_id id = it.typeId();
+    if( id == "paper" ) {
+        return -460; // -273C = no freeze
+    }
+    if( id == "ammonia" ) {
+        return -108; // -78C
+    }
+    if( id == "bleach" ) {
+        return 17; // -10C (assumption 8% sodium hypochlorite solution)
+    }
+    if( id == "salt_water" || id == "saline" ) {
+        return 28; // -2C (https://oceanservice.noaa.gov/facts/oceanfreeze.html)
+    }
+    std::string action = it.type->use_methods.begin()->first;
```

Comment:
Action-based freezing temperature isn't a very good idea. All alcohol drinks are made of `alcohol` (I mean the `material`).

Reply:
Stronger alcohols have lower freezing point. Vodka freezes in lower temperature then beer.

Reply:
I wasn't quite precise. Low alcohol drinks consist of `alcohol` and also `water`. You'd get exactly what you want by taking an average of their freezing temps.

Reply:
Ok, will certainly try to do that if can achieve exact same outcome. Thanks for the idea.